src/webdash.ts

```diff
@@ -113,19 +100,10 @@ class WebDash {
       "/"
     );
 
-    await this.worker_manager.executeWithAnyResponse(
-      dedent`
-        ${bootstrap_python}
-
-        # Initialise and bootstrap the dash app
-        app = bootstrap_dash_app("${url_base_pathname}")
-
-        @app.server.errorhandler(ImportError)
-        def import_error(err):
```

Comment:
Why are we no longer supporting lazy loading? The other side of the lazy loading seems to be untouched

Reply:
I wasn't aware of the other side. However, it did not work. To make it work we need `micropip.install` instead of just `__import__` (see new commit).